### PR TITLE
Solo bench

### PR DESCRIPTION
**Pull request by Nimfar11**
## About the PR

Solo bench with backrest. We can build. Throwing in a separate PR, as these were conceived without smoothing. Sofas and benches with smoothing will be in the past PR. 

## Why / Balance

For a variety of furniture

## Media

изображение

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

:cl: Nimfar11

- add: Adds solo comfy benches in different colors.

### CODE REVIEW

File: Resources/Prototypes/Entities/Structures/Furniture/bench.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
+- type: entity
+  name: comfortable bench
+  id: BenchComfSolo
+  parent: SeatBase
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Structures/Furniture/bench.yml:4]: Just call it comfy we don't pay by the character.

File: Resources/Prototypes/Entities/Structures/Furniture/bench.yml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  - type: Rotatable
+  - type: Sprite
+    sprite: Structures/Furniture/Bench/comf_bench.rsi
+    layers:
+    - state: bench_solo_base
+      color: "#767e82"
+    - state: bench_solo_cover
+  - type: Physics
+    bodyType: Static
+    canCollide: true
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Structures/Furniture/bench.yml:18]: Isn't this already default to true.

Comment (Nimfar11) [reply Resources/Prototypes/Entities/Structures/Furniture/bench.yml:18]: Tests in the past PR swear by it until I set it up like this.

File: Resources/Prototypes/Entities/Structures/Furniture/bench.yml (hunk 1 above)
Comment (Nimfar11) [reply Resources/Prototypes/Entities/Structures/Furniture/bench.yml]: done

File: Resources/Prototypes/Entities/Structures/Furniture/bench.yml [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
```
Comment (AJCM-git) [inline Resources/Prototypes/Entities/Structures/Furniture/bench.yml:18]: Pretty sure the test fail must've been erroneous so this can be safely removed

**Review by AJCM-git — CHANGES_REQUESTED** [review]
Idk, just adding 2 colors to construction is kind of limiting and not worth the boilerplate, probably best to just add a single color for now until we have better systems for coloring or randomize every time you make one (prob very annoying)

**Comment by AJCM-git** [comment]
> I can remove all colours except red and white (maybe leave blue and remove white). This was done primarily so that mappers or players could create multiple compositions of benches of the same colour. And random for mappers to be able to create variety.

You sure you pushed everything? Rn there are only 2 colors and base
I'm fine with having a entity prototype with different colors for mappers, the issue arises with construction since changing colors is incredibly boilerplatey, so for now just leave a single construction graph for either white or red

**Comment by Nimfar11** [comment]
Only red for construction. White (may look organically in the medical department) and multicoloured for mappers.

**Pull request merged**